packages/governance/package.json

```diff
@@ -78,4 +79,4 @@
   "typeCoverage": {
     "atLeast": 89.49
   }
-}
+}
```

Comment:
Is json happy with files that don't end with a linefeed?

Reply:
Yes, but git and editor software definitely prefer its presence.